VirtoCommerce.Storefront/Domain/Cart/Demo/CartConverter.cs

```diff
@@ -0,0 +1,73 @@
+using System;
+using System.Linq;
+using VirtoCommerce.Storefront.Model.Cart;
+using VirtoCommerce.Storefront.Model.Cart.Demo;
+using VirtoCommerce.Storefront.Model.Common;
+using cartDto = VirtoCommerce.Storefront.AutoRestClients.CartModuleApi.Models;
```

Comment:
cartApiDto

Reply:
Same naming style as in other parts of code